e2etest/native_staking_test.go

```diff
@@ -99,18 +99,18 @@ func TestNativeStaking(t *testing.T) {
 		cand2Addr := identityset.Address(1)
 		cand2PriKey := identityset.PrivateKey(1)
 
-		register1, err := action.SignedCandidateRegister(1, candidate1Name, cand1Addr.String(), cand1Addr.String(),
-			cand1Addr.String(), selfStake.String(), 91, true, nil, gasLimit, gasPrice, cand1PriKey)
-		require.NoError(err)
-		register2, err := action.SignedCandidateRegister(1, candidate2Name, cand2Addr.String(), cand2Addr.String(),
-			cand2Addr.String(), selfStake.String(), 1, false, nil, gasLimit, gasPrice, cand2PriKey)
-		require.NoError(err)
-
-		fixedTime := time.Unix(cfg.Genesis.Timestamp, 0)
-		require.NoError(ap.Add(context.Background(), register1))
-		require.NoError(createAndCommitBlock(bc, ap, fixedTime))
-		require.NoError(ap.Add(context.Background(), register2))
-		require.NoError(createAndCommitBlock(bc, ap, fixedTime))
+		addOneTx := func(tx action.SealedEnvelope, err error) action.SealedEnvelope {
```

Comment:
also return the error
`func(tx action.SealedEnvelope, err error) (action.SealedEnvelope, error)`